pkg/client/dynamic/clientset/tekton/tekton.go

```diff
@@ -2,20 +2,27 @@ package tekton
 
 import (
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
+	"github.com/tektoncd/triggers/pkg/apis/triggers"
 	"github.com/tektoncd/triggers/pkg/client/dynamic/clientset"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/client-go/dynamic"
 )
 
-var allowedTektonTypes = map[string][]string{
-	"v1alpha1": {"pipelineresources", "pipelineruns", "taskruns", "pipelines", "clustertasks", "tasks", "conditions"},
-	"v1beta1":  {"pipelineruns", "taskruns", "pipelines", "clustertasks", "tasks"},
-}
+var (
+	allowedPipelineTypes = map[string][]string{
+		"v1alpha1": {"pipelineresources", "pipelineruns", "taskruns", "pipelines", "clustertasks", "tasks", "conditions"},
+		"v1beta1":  {"pipelineruns", "taskruns", "pipelines", "clustertasks", "tasks"},
+	}
+	allowedTriggersTypes = map[string][]string{
+		"v1alpha1": {"clusterinterceptors"},
```

Comment:
Do we need to add `"clustertriggerbindings", "eventlisteners", "triggerbindings", "triggers", "triggertemplates"` here ?
/cc @dibyom

Reply:
I can for completeness, but given the way this access control works, it is not needed to pass tests.  You only need the exact versions through which the code accesses things.